Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -248,6 +272,20 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::UpdateToolsMenu(bool emulation_started)
+{
+  const bool enable_wii_tools = !emulation_started || !Settings::Instance().IsWiiGameRunning();
+  m_perform_online_update_menu->setEnabled(enable_wii_tools);
+  if (enable_wii_tools)
+  {
+    IOS::HLE::Kernel ios;
+    const auto tmd = ios.GetES()->FindInstalledTMD(0x0000000100000002);
```

Comment:
Sounds like a good idea, not everyone knows those long title ids by heart. And a good name always wins over a magic number in my opinion.